tools/pipelines/build-client.yml

```diff
@@ -99,16 +99,3 @@ extends:
           cp repo-package.json packages/package.json
           cp repo-package-lock.json packages/package-lock.json
 
-    additionalTests:
```

Comment:
I'd recommend removing this param from the template

Reply:
i can do that is a follow up, as this branch is annoying as it's not in my fork, and i block pushing to upstream from my local client, as i'm admin and can break stuff on accident